src/components/TextField/TextField.scss

```diff
@@ -29,7 +29,7 @@ $stacking-order: (
   flex-wrap: wrap;
 
   > .Input {
-    overflow: auto;
+    overflow: hidden;
```

Comment:
Not sure why but I think because the effects run after the function has run, the Textarea scrollbar was flickering when multiline.

Reply:
Looks like this won't work. The height is set based on the Resizer size, there's always a delay and the TextField scrolling is janky. Needed to add useLayoutEffect in the Resizer.